components/impls/core/src/main/javascript/components/PopupComponent/amd/ExportPopupComponent.js

```diff
@@ -364,13 +364,20 @@ define([
       var holderDiv = $('<div class="exportChartMasterDivHolder">');
       holderDiv.append(masterDiv);
 
-      $.fancybox({
+      $.fancybox.open({
+        src: holderDiv,
         type: "html",
-        closeBtn: true,
-        content: holderDiv,
-        width: totalWidth,
-        height: this.chartComponent.chartDefinition.height + 60,
-        autoDimensions: false
+        baseClass: "cdf-fancybox cdf-fancybox-html",
+        btnTpl: {
+          smallBtn:
+              '<button type="button" data-fancybox-close class="fancybox-button fancybox-close-small" title="close">' +
```

Comment:
As said in CDF's PR. This could be an SVG image.

Reply:
changed to a png file